wpeverest/user-registration
# UR-93 Feature- Getting started page

### All Submissions:






### Changes proposed in this Pull Request:
Previously after the first installation of our plugin, there was no onboarding page. This PR introduces Onboarding on the first installation of our plugin.

### How to test the changes in this Pull Request:

1. Setup this branch on a site where the user registration plugin is about to be installed for the first time.
2. On plugin installation you should be directed to the OnBoarding page.
3. Verify if the page is working properly.
4. Since all the configs related to Gutenberg blocks have been changed, please do verify if our blocks are working properly.

### Types of changes:




### Other information:




### Changelog entry

> Feature- Getting started page.
